libpod/container_internal_linux.go

```diff
@@ -1936,7 +1936,7 @@ func (c *Container) generatePasswdAndGroup() (string, string, error) {
 
 			f, err := os.OpenFile(containerPasswd, os.O_APPEND|os.O_WRONLY, 0600)
 			if err != nil {
-				return "", "", errors.Wrapf(err, "error opening container %s /etc/passwd", c.ID())
+				return "", "", errors.Wrapf(err, "container %s", c.ID())
```

Comment:
We might want to keep /etc/passwd in here given how long the path in the error is going to be

Reply:
Well my goal is to shorten the error messages and mentioning /etc/passwd twice and a bunch of other text does not help in my opinin.